docs/DesignDocs/ExtendedEnclaveInitializationData_V2.md

```diff
@@ -0,0 +1,182 @@
+Extended Enclave Initialization Data V2
+=======================================
+
+In this paper, we present an updated design of the extended enclave initialization data (EEID).
+
+Problem Statement
+-----------------
+
+In the current design of the EEID support (URL), the variable length configuration data buffer is part of the EEID pages added into the enclave by the enclave loader, and the enclave code access the configuration data through the eeid data structure. Intel's 10th-gen Core processor supports the Key Separation and Sharing (KSS) feature, including `CONFIGID` and `CONFIGSVN`, which are new fields defined in `SECS`. `CONFIGID` is intended to allow enclave creator to indicate what additional content may be accepted by the enclave post enclave initialization, whose exact usage depends on the enclave implementation. `CONFIGSVN` can be used in case `CONFIGID` does not fully reflect the identity of the additional content that may be accepted by the enclave post enclave initialization, for example, `CONFIGID` as the hash of the signing key for verifying the additional content. The `CONFIGID` and `CONFIGSVN` are part of the enclave identity produced by the CPU, and reflects the identity of the additional code/data allowed to be loaded into the enclave, committed at the enclave initialization time. The CONFIGID/CONFIGSVN based solution on SGX CPUs would expose a API set different from the current EEID implementation.
```

Comment:
Please break source lines into short lines, say 80 characters or less, to stick with OE style and make github diffs easier to review.  As it is, this is hard to review (especially after various suggestions like below where it's really hard to see what's changed in the suggestions).

Reply:
+1. I hoped that GitHub would show my suggestion diffs in a better way, but it doesn't. Best to stick to a line limit to keep diffs readable.

Reply:
Addressed